stripe/_list_object.py

```diff
@@ -50,6 +71,9 @@ def list(
             ),
         )
 
+    @_util.deprecated(
+        "This will be removed in a future version of stripe-python. Please call the `create` method on the corresponding resource directly, instead of using `create` from the list object."
+    )
     def create(
```

Comment:
I didn't realize we are removing an entire pattern. Hm.

Reply:
Yes I think we should discourage their use because
1. They aren't guaranteed to point to API methods that actually exist.
2. We can't give them specific types.